CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+
```

Comment:
A couple of nits with this description:

* The issue is sort of a leading backslash, but really it's a leading speech mark that is escaped by a backslash.
* There's a stray blank line at the top of the file (that shouldn't affect the CI/CD trying to create the release notes, but I'd get rid of it just to be safe).
*  `This causes the errors`, maybe instead: `This causes errors`.
* There's a missing full stop at the end of the paragraph.

I don't think any of these are showstoppers. Just me being a bit pedantic. (I tested the `bin/extract-release-notes.sh` script with a blank line at the top of `CHANGELOG.md` and it worked, but had the blank line that might look a bit odd)